staker/stakerapp.go

```diff
@@ -669,6 +676,102 @@ func (app *StakerApp) checkTransactionsStatus() error {
 			// we crashed after succesful send to babaylon, restart checking for unbonding signatures
 			app.wg.Add(1)
 			go app.checkForUnbondingTxSignaturesOnBabylon(stakingTxHash)
+		} else if localInfo.stakingTxState == proto.TransactionState_DELEGATION_ACTIVE {
+			// delegation was send to Babylon and activated by covenants, check wheter we:
+			// - did not spent tx beore restart
+			// - did not sent unbonding tx before restart
+			stakingTxHash := localInfo.stakingTxHash
+			tx, _ := app.mustGetTransactionAndStakerAddress(stakingTxHash)
+
+			// 1. First check if staking output is still unspent on BTC chain
+			stakingOutputSpent, err := app.wc.OutputSpent(stakingTxHash, tx.StakingOutputIndex)
+
+			if err != nil {
+				return err
+			}
+
+			if !stakingOutputSpent {
+				continue
```

Comment:
we can add a comment here explaining we don't need to do anything if the output is not spent